.gitignore

```diff
@@ -1,4 +1,8 @@
 *.log
 .p4config
-doc/html
+docs/html/
```

Comment:
Should these just be moved to the `docs/.gitignore` file?

Or better yet, can we move all of the generated doc files into a single directory that we can exclude without having to cherry-pick individual files/subdirs? Eg. `docs/build` or something.

Reply:
I don't think that's doable with how Jekyll works. It wants to serve up things in this directory. Perhaps it has support for out of tree builds that I'm unaware of though.

Reply:
Sure, I'll move these into docs/.gitignore.

Reply:
Moved all docs related .gitignores to `docs/.gitignore` in 0a924a7d.